tests/strategies/test_strategy_location_input.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+
+from koswat.dike_reinforcements.reinforcement_profile.standard.soil_reinforcement_profile import (
+    SoilReinforcementProfile,
+)
+from koswat.strategies.strategy_input import StrategyInput
+from koswat.strategies.strategy_reinforcement_type_costs import (
+    StrategyReinforcementTypeCosts,
+)
+
+
+class TestStrategyLocationInput:
+    def test_cheapest_reinforcement(self, example_strategy_input: StrategyInput):
+        # 1. Define test data.
+        _location_input = example_strategy_input.strategy_locations[0]
```

Comment:
Tip: Assert the expectations of this specific location, (perhaps create a fixture for this test class as you are only accessing this `location_input`.